cloud_modules/k8s.yml

```diff
@@ -4,7 +4,7 @@
   gather_facts: false
   tasks:
     - name: Get a list of all pods from any namespace
-      k8s_info:
+      community.kubernetes.k8s_info:
         kind: Pod
         namespace: "{{ namespace }}"
       register: pod_list
```

Comment:
In the next task, it uses the un-qualified `json_query`, and this gives problems with Ansible 2.10 and higher. That should come from a collection too, `community.general.json_query`

```
PLAYBOOK: k8s.yml **************************************************************
1 plays in cloud_modules/k8s.yml

PLAY [all] *********************************************************************
META: ran handlers

TASK [Get a list of all pods from any namespace] *******************************
task path: /tmp/awx_338_trs1vl89/project/cloud_modules/k8s.yml:6
ok: [HostConclusionWatch7679976248228587346949764062858367666829612043191587186] => {"changed": false, "resources": []}

TASK [Print pod names] *********************************************************
task path: /tmp/awx_338_trs1vl89/project/cloud_modules/k8s.yml:12
fatal: [HostConclusionWatch7679976248228587346949764062858367666829612043191587186]: FAILED! => {"msg": "template error while templating string: no filter named 'json_query'. String: pod_list: {{ pod_list | json_query('resources[*].metadata.name')  }} "}

PLAY RECAP *********************************************************************
HostConclusionWatch7679976248228587346949764062858367666829612043191587186 : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
```

Reply:
Alternatively, is it necessary to use `json_query`? I know in the past it's given people trouble since the library it depends on is, I believe, unmaintained. I've used different methods of getting the data I need out of returned info.

Reply:
Probably not - we could probably rewrite this to do something else.